### PR TITLE
refractor(balance_sheet) OHADA balance sheet

### PR DESCRIPTION
**Pull request by jeremielodi**

### CODE REVIEW

**Review by mbayopanda — COMMENTED** [review]
As a first implementation, the report looks good.

But i noticed that you've forgot to insert equity accounts in the report

File: server/controllers/finance/reports/balance_sheet/report.handlebars [hunk 1]
```diff
@@ -1,158 +1,106 @@
...
+            </td>
+            <td>
+              <table class="table table-condensed table-report table-bordered">
+                <thead>
+                  <th>{{translate 'FORM.LABELS.ACCOUNT_NUMBER'}}</th>
+                  <th>{{translate 'FORM.LABELS.LABEL'}}</th>
+                  <th>{{translate 'FORM.LABELS.AMOUNT'}}</th>
+                </thead>
+                <tbody>
+                  {{#each this.liability}}
```
Comment (mbayopanda) [inline server/controllers/finance/reports/balance_sheet/report.handlebars:58]: Equity accounts must appear above liabilities.

File: server/controllers/finance/reports/balance_sheet/index.js [hunk 2]
```diff
@@ -1,240 +1,153 @@
...
+  let docReport;
+  const options = _.extend(req.query, {
+    filename : 'TREE.OPERATING_ACCOUNT',
+    csvKey : 'rows',
+    user : req.session.user,
+  });
 
   try {
-    report = new ReportManager(TEMPLATE, req.session, params);
+    docReport = new ReportManager(TEMPLATE, req.session, options);
```
Comment (mbayopanda) [inline server/controllers/finance/reports/balance_sheet/index.js:33]: A curiosity question, is there any reason to rename this variable to `docReport` ?

File: server/models/Update/equity_accounts.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ UPDATE account SET type_id = 3
+ WHERE number LIKE '1%' AND type_id <> 6;
```
Comment (mbayopanda) [inline server/models/Update/equity_accounts.sql:2]: Could you do the same operation with data for assets and liabilities, to be sure that all accounts have the correct type.

It is important for correctly test what we have in our Bilan

**Review by mbayopanda — COMMENTED** [review]
It's look much better.

Could you fix the problem with tests

File: server/models/Update/equity_accounts.sql (hunk 3 above)
Comment (jeremielodi) [reply server/models/Update/equity_accounts.sql:2]: I need the assets and liabilities conditions for updating those accounts

Comment (mbayopanda) [reply server/models/Update/equity_accounts.sql:2]: You can use these document for seeing which account is for assets (actifs) and liabilities (passifs)

## Assets : 
actifs

## Liabilities
passifs

**Comment by jniles** [comment]
As discussed, closing this PR.

**Pull request closed without merging**